onprc_ehr/resources/web/onprc_ehr/model/sources/BiopsyNotes.js

```diff
@@ -16,10 +16,14 @@ EHR.model.DataModelManager.registerMetadata('Biopsy_Notes', {
 
         'study.encounters': {
             project: {
-                xtype: 'ehr-projectentryfield'
+                xtype: 'onprc_ehr-projectentryfield'
             }
         },
 
-
+        'study.pathologyDiagnoses': {
+            codesRaw: {
+                header: 'Snomed Codes'
```

Comment:
Is this intentional? If so, no problem, it's just that it's unrelated to the rest of the changes.

Reply:
Thanks, Josh.  Yes, the header as it appears in this code is what the customer requested to be displayed.